napari/layers/pyramid/pyramid.py

```diff
@@ -1,4 +1,6 @@
 import numpy as np
+
+from napari.util.status_messages import status_format
```

Comment:
relative path?

Reply:
Ah yes, good point, PyCharm does these imports automatically and I forgot to update this one.

Drives me nuts that they can't do relative imports!!! I might write a support ticket.